### PR TITLE
created a navbar and heroComponenet

### PR DESCRIPTION
**Pull request by iamluqmaan**
## List of changes
_Provide an overview_

## Screenshots
_Add before change and after change if it has been worked on previously_

## Steps on how to test it
_For example_
- Navigate to `/home` page
- Click on the about button
- It naviagtes to the `/about` page

### CODE REVIEW

**Comment by iamluqmaan** [comment]
I'm not done working on the hover states of the navbar but that's what I've covered so far.

File: package.json
Comment (AOMuiz) [inline package.json:1]: you shouldn't install any external package and if necessary consult first if its what we need.

File: src/components/navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    </nav>
+  )
+}
+
+
+
+
+// import React from "react";
+// import {Navbar, NavbarBrand, NavbarContent, NavbarItem, Link, Button, DropdownItem, DropdownTrigger, Dropdown, DropdownMenu} from "@nextui-org/react";
+// import {ChevronDown, Lock, Activity, Flash, Server, TagUser, Scale} from "./Icons.jsx";
```
Comment (AOMuiz) [inline src/components/navbar.tsx]: make sure to remove any unused code when done.

File: tailwind.config.ts [hunk 2]
```diff
@@ -47,6 +51,8 @@ const config: Config = {
         },
       },
     },
+    darkMode: "class",
+    plugins: [nextui()],
```
Comment (AOMuiz) [inline tailwind.config.ts]: remove code.

**Review by AOMuiz — CHANGES_REQUESTED** [review]

**Review by AOMuiz — APPROVED** [review]

**Pull request merged**